src/parseChangelog.ts

```diff
@@ -95,10 +108,13 @@ function parseChangelog({ changelogContent, repoUrl }) {
         throw new Error(`Malformed category header: '${truncated(line)}'`);
       }
       const isFirstCategory = mostRecentCategory === null;
-      finalizePreviousChange({ removeTrailingNewline: !isFirstCategory });
-      mostRecentCategory = results[1];
+      finalizePreviousChange({
+        removeTrailingNewline: !isFirstCategory,
+      });
+      // TODO: Get rid of typecast?
+      mostRecentCategory = results[1] as ChangeCategory;
```

Comment:
Curious about your thoughts here. I don't think we can be certain that this will be a valid change category?